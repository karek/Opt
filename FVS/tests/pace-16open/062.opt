7
42 13 43 31 46 19 28 
