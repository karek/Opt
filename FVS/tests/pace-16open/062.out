7
40 28 43 31 25 46 54 
